src/network/package.h

```diff
@@ -70,6 +78,6 @@ i32 format(char *buffer, u32 size, FormatHint args, Package pkg);
 
 #ifndef IMGUI_DISABLE
 void imgui_package_create(Package *package, WipEntities *wip_entities);
-void imgui_event_create(Package *package, WipEntities *wip_entities);
+void imgui_event_create(Event *event, WipEntities *wip_entities);
 void imgui_package_show(Package *package);
 #endif
```

Comment:
Maybe stub these if IMGUI_DISABLE is defined. Instead of commenting out the definitions. Be the change you want to see in the world.

Reply:
I think it's better to do all these at the same time instead of differing behavior for different parts.

Reply:
(#122)